### PR TITLE
Improved french translation

### PR DESCRIPTION
**Pull request by jeluard**
### Summary:

Update French translation to match latest labels

status: ready

### CODE REVIEW

File: src/status_im/translations/fr.cljs [hunk 1]
```diff
@@ -249,7 +262,7 @@
    :confirm                               "Confirmer"
    :transaction                           "Transaction"
    :unsigned-transaction-expired          "La transaction non signée a expiré"
-   :status                                "Statut"
+   :status                                "Status"
```
Comment (mratsim) [inline src/status_im/translations/fr.cljs:265]: It should be "Statut" previous version was correct (unless it's intended for Status.im)

Comment (jeluard) [reply src/status_im/translations/fr.cljs:265]: It's the app name

File: src/status_im/translations/fr.cljs [hunk 2]
```diff
@@ -287,9 +300,14 @@
    :wallet-add-asset                      "Ajouter un actif"
    :wallet-total-value                    "Valeur totale"
    :wallet-settings                       "Paramètres du portefeuille"
+   :wallet-manage-assets                  "Gérer les actifs"
    :signing-phrase-description            "Signez la transaction en saisissant votre mot de passe. Assurez-vous que les mots ci-dessus correspondent à votre phrase de signature secrète"
    :wallet-insufficient-funds             "Fonds insuffisants"
    :request-transaction                   "Effectuer une transaction"
+   :receive-transaction                   "Recevoir une transaction"
+   :new-transaction                       "Nouvelle Transaction"
+   :transaction-history                   "Historique de transactions"
```
Comment (mratsim) [inline src/status_im/translations/fr.cljs]: "Historique des transactions"

File: src/status_im/translations/fr.cljs [hunk 3]
```diff
@@ -287,9 +300,14 @@
    :wallet-add-asset                      "Ajouter un actif"
    :wallet-total-value                    "Valeur totale"
    :wallet-settings                       "Paramètres du portefeuille"
+   :wallet-manage-assets                  "Gérer les actifs"
    :signing-phrase-description            "Signez la transaction en saisissant votre mot de passe. Assurez-vous que les mots ci-dessus correspondent à votre phrase de signature secrète"
    :wallet-insufficient-funds             "Fonds insuffisants"
    :request-transaction                   "Effectuer une transaction"
+   :receive-transaction                   "Recevoir une transaction"
+   :new-transaction                       "Nouvelle Transaction"
```
Comment (mratsim) [inline src/status_im/translations/fr.cljs:308]: What's the difference between a "transaction request" and a "new transaction" in English?

File: src/status_im/translations/fr.cljs [hunk 4]
```diff
@@ -337,6 +355,7 @@
    :add-network                           "Ajouter un réseau"
    :add-new-network                       "Ajouter un nouveau réseau"
    :existing-networks                     "Réseaux existants"
+   :existing-wnodes                       "Serveurs de mail existants"
    :add-json-file                         "Ajouter un fichier JSON"
    :paste-json-as-text                    "Coller JSON en texte"
```
Comment (mratsim) [inline src/status_im/translations/fr.cljs]: "Coller un JSON comme texte"

File: src/status_im/translations/fr.cljs [hunk 5]
```diff
@@ -337,6 +355,7 @@
...
    :paste-json                            "Coller JSON"
```
Comment (mratsim) [inline src/status_im/translations/fr.cljs]: "Coller un JSON"

File: src/status_im/translations/fr.cljs [hunk 6]
```diff
@@ -346,10 +365,17 @@
    :process-json                          "Interpréter le JSON"
    :error-processing-json                 "Erreur survenue lors de l'interprétation du JSON"
    :rpc-url                               "URL RPC"
+   :network                               "Réseau"
    :remove-network                        "Supprimer le réseau"
+   :offline-messaging-settings            "Configuration messagerie hors-ligne"
```
Comment (mratsim) [inline src/status_im/translations/fr.cljs]: Configuration de la messagerie hors-ligne

File: src/status_im/translations/fr.cljs [hunk 7]
```diff
@@ -28,6 +30,7 @@
    ;;drawer
    :switch-users                          "Changer d'utilisateur"
    :current-network                       "Le réseau actuel"
+   :logout                                "Se déloguer"
```
Comment (mratsim) [inline src/status_im/translations/fr.cljs]: "Déconnexion"

File: src/status_im/translations/fr.cljs [hunk 8]
```diff
@@ -28,6 +30,7 @@
    ;;drawer
    :switch-users                          "Changer d'utilisateur"
    :current-network                       "Le réseau actuel"
```
Comment (mratsim) [inline src/status_im/translations/fr.cljs]: "Réseau actuel"

**Review by mratsim — COMMENTED** [review]
Thanks for the translation, I've added a few pronouns that makes it flowing better in my opinion.

File: src/status_im/translations/fr.cljs (hunk 3 above)
Comment (jeluard) [reply src/status_im/translations/fr.cljs:308]: Not used in same context. Action vs title

**Review by flexsurfer — APPROVED** [review]

File: src/status_im/translations/fr.cljs (hunk 7 above)
Comment (yenda) [reply src/status_im/translations/fr.cljs]: J'avoue se déloguer c'est horrible :D

**Comment by jeluard** [comment]
@yenda @mratsim Updated. Please accept the PR!

**Review by mratsim — APPROVED** [review]

**Review by yenda — APPROVED** [review]

**Pull request merged**